DisguiseUnityRenderStream/Runtime/Util/RegistryWrapper.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Disguise.RenderStream.Utils
+{
+    // https://www.pinvoke.net/default.aspx/advapi32/regopenkeyex.html
+    static class RegistryWrapper
+    {
+        public static UIntPtr HKEY_LOCAL_MACHINE = new UIntPtr(0x80000002u);
+        public static UIntPtr HKEY_CURRENT_USER  = new UIntPtr(0x80000001u);
+
+        public enum ReadRegKeyResult
+        {
+            Success,
+            OpenFailed,
+            QueryValueFailed,
+            TypeNotSupported
+        }
+
+        // Only supports REG_SZ registry value type
+        public static ReadRegKeyResult ReadRegKey(UIntPtr rootKey, string keyPath, string valueName, out string value)
+        {
+            value = null;
+            
+            if (Native.RegOpenKeyEx(rootKey, keyPath, 0, Native.KEY_READ, out var hKey) == 0)
```

Comment:
Minor: Instead of == 0, you could do == ERROR_SUCCESS (and define it below as you do with REG_NONE and REG_SZ).